README.md

```diff
@@ -213,26 +278,26 @@ As already discussed, employees can get restricted stock, stock options, or RSUs
 ### 83(b) elections
 
-- Generally, restricted stock is taxed [when it vests](http://www.investopedia.com/articles/tax/09/restricted-stock-tax.asp?performancelayout=true) as ordinary income. Of course, if the stock is in a startup with low value, this may not result in very much tax being owed. But if it's been years since the stock was first granted and the company is now worth a lot, the taxes owed could be significant.
-- 🔹 However, the Internal Revenue Code offers an alternative, called a [**Section 83(b) election**](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/), which is an election to be taxed on the receipt of the property, even though you might not get to keep it since it has not vested. The presumption of the tax law would normally be that you do not owe tax until property you have received vests. With a Section 83(b) election, you’re telling the IRS you want to pay taxes early, on stock that is not yet vested, instead of paying as it vests. The election can potentially reduce your tax significantly: If the shares go up in value, the taxes owed on vesting might be far greater than the taxes owed at the time of receipt.
-    - An 83(b) election isn’t guaranteed to reduce your taxes, of course. For example, the value of the stock may not increase. And if you leave the company before you vest, you don't get the taxes you’ve paid back.
+- Generally, restricted stock is taxed as ordinary income *[when it vests](http://www.investopedia.com/articles/tax/09/restricted-stock-tax.asp?performancelayout=true).* If the stock is in a startup with low value, this may not result in a high tax. If it’s been years since the stock was first granted and the company is now worth a lot, the taxes owed could be quite significant.
+- 🔹 The Internal Revenue Code offers an alternative that will protect you from this high potential tax. It’s called a [**Section 83(b) election**](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/), and ensures that you’ll be taxed on the receipt of the “property” (the stock), rather than at the time the stock vests. Basically, with a Section 83(b) election, you’re telling the IRS you want to pay taxes early, and this can potentially reduce your tax significantly: If the shares go up in value, the taxes owed on vesting might be far greater than the taxes owed at the time of receipt.
+    - An 83(b) election isn’t guaranteed to reduce your taxes, of course. For example, the value of the stock may not increase. And if you leave the company before you vest, you *don’t* get the taxes you’ve paid back.
     - ❗ You must file the 83(b) election yourself with the IRS [**within 30 days**](https://www.irs.gov/irb/2012-28_IRB/ar12.html) of the grant or exercise, or the opportunity is irrevocably lost.
     - 🔸 Section 83(b) elections cannot be made on the receipt of a stock option. They can only be made on the receipt of actual shares of stock.
```

Comment:
Huh? But don't most employers grant ISOs or NSOs? If that's true, how can I file an 83b?

Reply:
Rewrote: 

  - ☝️ Note an 83(b) election is made receipt of actual shares of stock.
    Technically, it cannot be made on the receipt of a stock *option* itself:
    You first must exercise that option, then file the election.